params/chainspecs/sepolia.json

```diff
@@ -15,6 +15,7 @@
   "terminalTotalDifficulty": 17000000000000000,
   "terminalTotalDifficultyPassed": true,
   "mergeNetsplitBlock": 1735371,
+  "mergeBlock": 1735371,
```

Comment:
Should be 1450408: see https://sepolia.etherscan.io/block/1450408

Reply:
I believe it is https://sepolia.etherscan.io/block/1450409 - the first block that was "proposed on" a slot, and the first block after TTD has been reached.

PoW did 1450408, increasing TD past the threshold. Then the very next block is PoS

Reply:
Yes, you're right, the first PoS block is 1450409 on Sepolia.